Sources/RTVIClientIOSDaily/DailyVoiceClient.swift

```diff
@@ -6,6 +6,9 @@ import RTVIClientIOS
 public class DailyVoiceClient: RTVIClient {
     
     public init(baseUrl:String? = nil, options: RTVIClientOptions) {
+        if baseUrl == nil && options.params.baseUrl == nil {
+            print("DailyVoiceClient will not work without a baseUrl specified")
```

Comment:
`print()` isn't ideal, but currently `PipecatClientIOSDaily` doesn't include a `Logger` implementation. I didn't want to add all that yet just for this one log.

Reply:
Makes sense. 👍